pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -290,9 +294,27 @@ def _string_to_circuit(
     return circ
 
 
+def _get_pytket_ctrl_state(bitstring: str, n_bits: int) -> tuple[bool, ...]:
+    "Converts a little endian string '001'=1 (LE) to (1, 0, 0)."
```

Comment:
That looks like a big-endian string if the value is 1. But you are just reversing it. Could implement this as:
```python
    return tuple(reversed(list(map(int, bitstr))))
```
Don't need the `n_bits` parameter.

Reply:
The problem is that we don't know the number of control qubits in advance. 

We could have a binary encoding of "7" in 3 bits or 4 bits or a larger number. We still need to produce a list of the correct length to pass to `QControlBox`

Reply:
OK, in that case can implement as
```python
return tuple(reversed(list(map(int, bitstr.zfill(n_bits)))))
```
?

Reply:
The following works well for me (all tests pass) .

```python
def _get_pytket_ctrl_state(bitstring: str, n_bits: int) -> tuple[bool, ...]:
    assert set(bitstring).issubset({"0", "1"})
    new_bitstring = bitstring.zfill(n_bits)
    pytket_ctrl_state = reversed([bool(int(b)) for b in new_bitstring])
    return tuple(pytket_ctrl_state)
```

The snippet above didn't work as a drop in but I think its a case of the zeros being added to the right rather than the left.

I didn't know about `str.z_fill()` thanks.